core/object/object.h

```diff
@@ -101,6 +101,7 @@ enum PropertyHint {
 };
 
 enum PropertyUsageFlags {
+	PROPERTY_USAGE_NONE = 0,
```

Comment:
Sorry I forgot this in the previous review: I think that change makes sense but since it's not used in this PR, I'd leave it out here.

Could be worth having a PR that adds it and replaces uses of `0` by `PROPERTY_USAGE_NONE` I guess.